packages/simple-hub/jest/jest.chain.config.js

```diff
@@ -1,5 +0,0 @@
-const config = require('./jest.config');
```

Comment:
I wonder if this jest configuration should remain. We could have chain tests in the near future, and I don't see much harm in leaving the configuration if there are no tests that match the `testMatch` pattern. Of course, to play devil's advocate, we shouldn't have any dead code in the repo.

Reply:
One reason I deleted it was, the `chain:test` command was failing but not the jest process was not exiting.

I've restored `jjest.chain.config.js` for now.

Reply:
>I don't see much harm in leaving the configuration if there are no tests that match the testMatch pattern.

Unfortunately, this results in a jest failure :P